Ships as Raw.txt

```diff
@@ -159,10 +160,11 @@ Nav chart
 Speed 1|1
 Speed 2|1 0 
 Speed 3|0 1 0
-Front Arc 4 |RRRKKK
-Side Arc 2 |RRK
-Rear Arc 2 |RBK
-Anti Squadron|KK
+Speed 4|-
+FrontArc 4 |RRRKKK
+SideArc 2 |RRK
+RearArc 2 |RBK

```

Comment:
You mentioned this in another issue, let's get these all changed to whatever we decide in that conversation thread.